index.php

```diff
@@ -1,14 +1,21 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
+ *
+ * @package WordPress
  */
 
+// Hook.
```

Comment:
@nylen Do you have a suggestion to what should go here? Leaving it without inline docs, feels weird. I appreciate your feedback.